i6_models/losses/nce.py

```diff
@@ -0,0 +1,105 @@
+__all__ = [
+    "NoiseContrastiveEstimationLossV1",
+]
+
+import torch
+from torch import nn
+from torch.nn import functional as F
+from typing import Optional
+
+
+class NoiseContrastiveEstimationLossV1(nn.Module):
```

Comment:
This class is a subclass of nn.Module, which makes it a part and thus should be in the parts folder instead of a dedicated samplers folder.